mantaray/received.py

```diff
@@ -319,15 +324,14 @@ def _handle_kick(server_view: views.ServerView, kicker: str, args: list[str]) ->
             [
                 views.MessagePart(kicker, tags=[kicker_tag]),
                 views.MessagePart(" has kicked you from "),
-                # TODO: use the channel tag more, make clickable?
+                # TODO: Make channel tag clickable?
                 views.MessagePart(channel_view.channel_name, tags=["channel"]),
                 views.MessagePart(
                     f". (Reason: {reason}) You can still join by typing "
                 ),
                 # TODO: new tag instead of abusing the "pinged" tag for this
-                views.MessagePart(
-                    f"/join {channel_view.channel_name}", tags=["pinged"]
-                ),
+                views.MessagePart("/join ", tags=["pinged"]),
+                views.MessagePart(channel_view.channel_name, tags=["channel"]),
```

Comment:
I don't like how the command now shows up as two differently tagged parts. How is the user supposed to know that `/join #foo` is a thing, if `/join` and `#foo` have separate colors and seem to be two different things?

I also think it's a good thing if the channel isn't clickable here. If you were kicked from a channel, you are probably in a heated argument, and typing `/join #channel` manually could give you time to calm down a bit.

I'd suggest leaving this part unchanged for now. We can later change the `"pinged"` tag easily.